arch/cortex-m4/src/lib.rs

```diff
@@ -264,125 +39,3 @@ pub unsafe extern "C" fn switch_to_user(
 ) -> *const usize {
     unimplemented!()
 }
-
-#[cfg(all(target_arch = "arm", target_os = "none"))]
-#[naked]
-pub unsafe extern "C" fn hard_fault_handler() {
-    let faulting_stack: *mut u32;
-    let kernel_stack: bool;
-
-    // First need to determine if this a kernel fault or a userspace fault.
-    llvm_asm!(
-    "
-    mov    r1, 0     /* r1 = 0 */
-    tst    lr, #4    /* bitwise AND link register to 0b100 */
-    itte   eq        /* if lr==4, run next two instructions, else, run 3rd instruction. */
-    mrseq  r0, msp   /* r0 = kernel stack pointer */
-    addeq  r1, 1     /* r1 = 1, kernel was executing */
-    mrsne  r0, psp   /* r0 = userland stack pointer */"
-    : "={r0}"(faulting_stack), "={r1}"(kernel_stack)
-    :
-    : "r0", "r1"
-    : "volatile" );
```

Comment:
Why are you removing this?

Reply:
It's removed in master... not sure why it's showing up.